CHANGELOG.md

```
@@ -1,76 +1,79 @@
 # Prowide Core - CHANGELOG
 
-#### 9.5.2 - SNAPSHOT
+#### 10.2.2 - SNAPSHOT
```

Markdownlint

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)